### PR TITLE
Added documentation for APM-173

### PR DESCRIPTION
**Pull request by cpjulia**
Documentation for https://github.com/arangodb/arangodb/pull/14830

### CODE REVIEW

File: 3.9/programs-arangobench.md [hunk 1]
```diff
@@ -26,7 +26,16 @@ The most important general _arangobench_ options are:
 
 - `--server.endpoint`: the server endpoint to connect to. This can be a remote
   server or a server running on the same host. The endpoint also specifies
-  whether encryption at transit (TLS) should be used.
+  whether encryption at transit (TLS) should be used. Multiple endpoints can be
+  provided. Example:
+ 
+  ```js
```
Comment (Simran-B) [inline 3.9/programs-arangobench.md:32]:
````suggestion
  ```
````

File: 3.9/release-notes-new-features39.md [hunk 2]
```diff
@@ -391,6 +391,21 @@ See [Overriding data types per attribute](programs-arangoimport-examples-csv.htm
 
 ### arangobench
 
+Now it supports multiple coordinators, so the flag `--server.endpoint` can be
+used multiple times, as in the example below:
```
Comment (Simran-B) [inline 3.9/release-notes-new-features39.md:395]:
```suggestion
_arangobench_ now supports multiple Coordinators. The flag `--server.endpoint`
can be specified multiple times, as in the example below:
```

File: 3.9/release-notes-new-features39.md [hunk 3]
```diff
@@ -391,6 +391,21 @@ See [Overriding data types per attribute](programs-arangoimport-examples-csv.htm
...
+
+```js
```
Comment (Simran-B) [inline 3.9/release-notes-new-features39.md:397]:
````suggestion
```
````

File: 3.9/release-notes-new-features39.md [hunk 4]
```diff
@@ -391,6 +391,21 @@ See [Overriding data types per attribute](programs-arangoimport-examples-csv.htm
...
+arangobench \
+  --server.endpoint tcp://[::1]::8529 \
+  --server.endpoint tcp://[::1]::8530 \
+  --server.endpoint tcp://[::1]::8531 
+``` 
+
+This does not compromise the use of the other client tools, which preserve
+the behavior of having one coordinator and one endpoint.
```
Comment (Simran-B) [inline 3.9/release-notes-new-features39.md:405]:
```suggestion
This does not compromise the use of the other client tools, that preserve
the behavior of having one Coordinator and one endpoint.
```

File: 3.9/release-notes-new-features39.md [hunk 5]
```diff
@@ -391,6 +391,21 @@ See [Overriding data types per attribute](programs-arangoimport-examples-csv.htm
...
+
+Refer to [arangobench options](programs-arangobench.html#general-configuration)
```
Comment (Simran-B) [inline 3.9/release-notes-new-features39.md:407]:
```suggestion
Also see [_arangobench_ Options](programs-arangobench.html#general-configuration)
```

**Review by Simran-B — APPROVED** [review]

File: 3.9/release-notes-new-features39.md [hunk 6]
```diff
@@ -391,6 +391,21 @@ See [Overriding data types per attribute](programs-arangoimport-examples-csv.htm
...
+
+```
+arangobench \
+  --server.endpoint tcp://[::1]::8529 \
+  --server.endpoint tcp://[::1]::8530 \
+  --server.endpoint tcp://[::1]::8531 
+``` 
+
+This does not compromise the use of the other client tools, that preserve
+the behavior of having one Coordinator and one endpoint.
```
Comment (jsteemann) [inline 3.9/release-notes-new-features39.md:405]: What about arangodump and arangorestore?

**Pull request merged**